includes/embeds/class-amp-instagram-embed.php

```diff
@@ -145,6 +146,7 @@ private function create_amp_instagram_and_replace_node( $dom, $node ) {
 
 		$new_node = AMP_DOM_Utils::create_node( $dom, $this->amp_tag, array(
 			'data-shortcode' => $instagram_id,
+			'data-captioned' => '',
```

Comment:
@westonruter Do we need to add this attribute only if `<blockquote>` element has `data-instgrm-captioned` attribute?

Reply:
Thanks, I've made changes and PR updated.